packages/webapp/src/components/TileDashboard/DashboardTile/styles.module.scss

```diff
@@ -0,0 +1,111 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+@import '../../../assets/mixin';
+
+.tile {
+  height: 40px;
+  width: 100%;
+  min-width: 90px;
+
+  display: flex;
+
+  justify-content: center;
+  align-items: center;
+  flex-shrink: 0;
+  gap: 8px;
+  padding: 0px 8px;
+
+  border-radius: 4px;
+  background: var(--Colors-Accent---singles-Icon-action, #2d5550);
```

Comment:
When inspecting the tiles in dev tools I see that this variable is not defined -- and although the fallback color matches Figma, in Figma there's an opacity of 25% applied. For reference this is what I'm seeing in Storybook (basically the tiles are blending with the background)

<img width="690" alt="Captura de pantalla 2024-02-15 a la(s) 10 13 26" src="https://github.com/LiteFarmOrg/LiteFarm/assets/20230771/10915605-8274-4541-8d34-7b144c5a00c9">

Reply:
Wait, sorry could you explain this one further?  I would see this over and over in Storybook, and every time it would be resolved by a restart of the Storybook server -- then the colours would return to normal! This is from an undefined CSS variable and the fallback only sometimes being applied? How does the opacity figure in?

Reply:
@antsgar after adding the new CSS colour this still happens for me all the time in Storybook 😞 If not CSS variable, I don't know what could be causing this!